### PR TITLE
[show] Add support for SONiC Gearbox Manager via new gearboxutil utility

### PR DESCRIPTION
**Pull request by sydlogan**
* add and modify command line utilities to support gearbox phy
* added build time mock unit tests

HLD is located at https://github.com/Azure/SONiC/blob/b817a12fd89520d3fd26bbc5897487928e7f6de7/doc/gearbox/gearbox_mgr_design.md

  Signed-off-by: syd.logan@broadcom.com

**- What I did**

**- How I did it**

**- How to verify it**

**- Previous command output (if the output of a command-line utility has changed)**

**- New command output (if the output of a command-line utility has changed)**

### CODE REVIEW

**Comment by sydlogan** [comment]
Test fails should go away once https://github.com/Azure/sonic-py-swsssdk/pull/79 is merged.

**Comment by sydlogan** [comment]
@jleveque Hi, would you be able to give an approving review to this pull request?

**Comment by sydlogan** [comment]
Added documentation for two new show commands.

**Comment by jleveque** [comment]
Please fix LGTM alert

File: doc/Command-Reference.md [hunk 1]
```diff
@@ -2257,6 +2259,56 @@ The list of the WRED profile fields that are configurable is listed in the below
...
+
+  ```
+
+**show gearbox phys status**
+
+This command displays basic information about the gearbox phys configured on the switch. 
+
+- Usage:
+  ```
+  show gearbox phy status
```
Comment (jleveque) [inline doc/Command-Reference.md:2296]: Here, the subgroup is `phy` (singular), whereas it is plural elsewhere. Our CLI supports abbreviations (prefix-matching), so this command should work, but I prefer to keep things consistent in the documentation.

Comment (sydlogan) [reply doc/Command-Reference.md]: good catch - commit fa9097f520e04dc3a0a5d8ebb3fe620137dce924

File: scripts/gearboxutil [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+    """
+    Get APPL_DB key attribute
+    """
+
+    val = db.get(db.APPL_DB, key, attr)
+    if val is None:
+        return "N/A"
+
+    if "speed" in attr:
+
```
Comment (jleveque) [inline scripts/gearboxutil:57]: Suggest to delete this blank line to align with prevailing style throughout

File: scripts/gearboxutil [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+                get_appl_key_attr(self.appl_db, GEARBOX_TABLE_PHY_PREFIX.format(phy_id), PHY_NAME),
+                get_appl_key_attr(self.appl_db, GEARBOX_TABLE_PHY_PREFIX.format(phy_id), PHY_FIRMWARE_MAJOR_VERSION))
+            table.append(data_row)
+
+        # Sorting and tabulating the result table.
+        sorted_table = natsorted(table)
+        print tabulate(sorted_table, phy_header_status, tablefmt="simple", stralign='right')
+
+    def __init__(self):
+
```
Comment (jleveque) [inline scripts/gearboxutil:130]: Suggest to delete this blank line to align with prevailing style throughout

File: scripts/gearboxutil [hunk 4]
```diff
@@ -0,0 +1,228 @@
...
+                get_appl_key_attr(self.appl_db, PORT_TABLE_ETHERNET_PREFIX.format(name), PORT_ADMIN_STATUS))
+
+            table.append(data_row)
+
+        # Sorting and tabulating the result table.
+        sorted_table = natsorted(table)
+        print tabulate(sorted_table, intf_header_status, tablefmt="simple", stralign='right')
+
+    def __init__(self):
+
```
Comment (jleveque) [inline scripts/gearboxutil:188]: Suggest to delete this blank line to align with prevailing style throughout

**Comment by sydlogan** [comment]
@jleveque thank you for the review and approval

**Pull request merged**